### PR TITLE
Renaming the tests

### PR DESCRIPTION
**Pull request by fly2abhishek**

### CODE REVIEW

File: src/Form/AutologoutSettingsForm.php [hunk 1]
```diff
@@ -248,21 +248,20 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
 
     if($values['role_logout']) {
       // Validate timeouts for each role.
-      foreach (user_roles(TRUE) as $key => $role) {
-        if (empty($new_stack[$key])) {
+      foreach (user_roles(TRUE) as $role => $role_object) {
+        if (empty($new_stack[$role]) || $new_stack[$role]['enabled'] == 0 ) {
```
Comment (naveenvalecha) [inline src/Form/AutologoutSettingsForm.php:252]: The other key changes are unexpected; the $role variable name looks better than $role object. The only change needed was $new_stack[$role]['enabled'] == 0
can we revert it ?

File: src/Tests/AutologoutTest.php [hunk 2]
```diff
@@ -178,7 +188,7 @@ public function testAutologoutDefaultTimeout() {
     $this->drupalGet('node');
     $this->assertResponse(200, 'Homepage is accessible');
     $this->assertNoRaw(t('Log out'), 'User is no longer logged in.');
-    $this->assertRaw(t('You have been logged out due to inactivity.'), 'User sees inactivity message.');
+    $this->assertText(t('You have been logged out due to inactivity.'), 'User sees inactivity message.');
```
Comment (naveenvalecha) [inline src/Tests/AutologoutTest.php:191]: nice catch :) we don't need the raw here

**Comment by naveenvalecha** [comment]
Nothing looks blocker and good to be in

File: src/Form/AutologoutSettingsForm.php (hunk 1 above)
Comment (fly2abhishek) [inline src/Form/AutologoutSettingsForm.php:252]: The data that we want here is the key of the array, hence named it as $role, which is used in Line no 260. $role_object is not really used any where.

**Pull request merged**